msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -536,6 +536,9 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 	</Target>
 
 	<Target Name="_DetectSdkLocations" DependsOnTargets="_ComputeTargetArchitectures">
+		<Error Text="Support for classic applications (using monotouch.dll) is obsoleted in Xamarin.iOS 10.0+"
```

Comment:
Yea I tried to come up with an other way to automatically present the migration to the user (so that message wouldn't be needed in the build error) but I don't see it (:

Having a more verbose message doesn't hurt anyway.